client/my-sites/plan-price/index.tsx

```diff
@@ -375,6 +385,23 @@ function MultiPriceDisplay( {
 	);
 }
 
+function CurrencySymbolDisplay( {
+	currencySymbol,
+	hasUslocaleInSymbol,
+}: {
+	currencySymbol: string;
+	hasUslocaleInSymbol: boolean;
```

Comment:
I think it might be better if we generalize this variable here and when it is called to something like `shouldBeSuperscripted` or something better. In this function we could have a mapping object that it could check to get the right markup for each currency if there is ever more than one 🤔 

However if we are certain this won't be a problem for any other currency, maybe this is overkill

Reply:
As mentioned in my comment above, the US$ currency symbol is a specific edge case unique to the USD currency only, so I don't think we want to add any additional markup to any of the other currency symbols.